tests/testthat/test-plot.R

```diff
@@ -0,0 +1,33 @@
+context("Plot")
+
+test_that("Dendrogram", {
+    dendro.obj <- sits_dendrogram (cerrado_2classes, bands = c("ndvi"))
+    sits_plot_dendrogram(cerrado_2classes, dendro.obj)
+    expect_true(TRUE)
+})
+
+test_that("All", {
```

Comment:
How to test 'plots'?

Reply:
I don´t know yet. For now, the tests only ensure that the plot commands run (but not if they run properly).